uax/uax42/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Generating TR42
+
+## Step 1 - Generate property value fragments
+
+- Run org.unicode.xml.GeneratePropertyValues to populate the UNICODETOOLS_REPO_DIR/uax/uax42/fragments/ folder.
+
+## Step 2 - Generate TR42 index.html and index.rnc 
+
+- In UNICODETOOLS_REPO_DIR/uax/uax42/ run `mvn xml:transform`
+
+  index.html and index.rnc will be generated in UNICODETOOLS_REPO_DIR/uax/uax42/output/
+
+## Step 3 - Validate generated UAX XML files
+
+You'll need a [RELAX NG](https://relaxng.org/) schema validator. We'll use [jing-trang](https://github.
+com/relaxng/jing-trang) in this example.
+
+1. Clone and build [jing-trang](https://github.com/relaxng/jing-trang)
+2. Run the following:
+    ```
+   java -jar C:\_git\jing-trang\build\jing.jar -c UNICODETOOLS_REPO_DIR\uax\uax42\output\index.rnc <path to UAX xml file>
+   ```
+   Note that the UAX xml file has to be saved as NFD as the Unihan syntax regular expressions are expecting NFD.
```

Comment:
Does the tool generate the data files in NFD?
It seems like the files should come out as needed for the tool chain as well as for publication.

Reply:
The data files are not NFD by default, and I'm not sure that we should change the format for publication.
The rest of your question ties back to my previous comment; we could add a step to create an NFD version of the UCDXML files as part of an end-to-end process. However, is that in scope?